src/main/cs/Core/Asegurados/Age.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace RI.Novus.Core.Asegurados;
+
+/// <summary>
+/// Represent asegurado's age.
+/// </summary>
+public sealed class Age : AbstractPositiveIntegerPrimitive
+{
+    private static readonly PositiveInteger MinValue = new(1);
+    private static readonly PositiveInteger MaxValue = new(int.MaxValue);
```

Comment:
```suggestion
    private static readonly PositiveInteger MaxValue = new(130);
```
Any human exists that manage to had int.MaxValue years